internal/ocm/upgradepolicy_test.go

```diff
@@ -20,17 +20,22 @@ func TestClientPatchUpgradePolicy(t *testing.T) {
 	s := httptest.NewServer(http.HandlerFunc(func(rw http.ResponseWriter, r *http.Request) {
 		recordedHttpRequest = r
 		recordedBody, _ = ioutil.ReadAll(recordedHttpRequest.Body)
-		fmt.Fprintln(rw, `{"response":"works!"}`)
+		if r.URL.Path == "/proxy/apis/api/clusters_mgmt/v1/clusters" {
+			fmt.Fprintln(rw, `{"items": [{"kind": "Cluster","id": "1ou","external_id": "123"}]}`)
+		} else {
+			fmt.Fprintln(rw, `{"response":"works!"}`)
+		}
 	}))
 	defer s.Close()
 
-	c := NewClient(
-		WithClusterID("123"),
+	ctx := context.Background()
+
+	c, _ := NewClient(
```

Comment:
should check `err` return value, instead discarding it.
`require.NoError(t, err)`